tests/testthat/test-helpers_jags.R

```diff
@@ -0,0 +1,230 @@
+# h_jags_add_dummy ----
```

Comment:
file name is correct? just double check

Reply:
@danielinteractive, yes, I think that `helpers_jags.R` is better than `jags.R` as these are still helper functions as we prefix them with `h_`. For instance we already have files likes `utils.R`, `mcmc.R` which are not helpers do I did not want to confuse it with just `jags.R`. But if you still prefer to keep `jags.R`, let me know please and I will change it :) Tx!